README.md

```diff
@@ -1,3 +1,15 @@
+---
+page_type: sample
+name: OpenXR Mixed Reality samples for Unity
+description: These sample projects showcase how to build Unity applications 
+for HoloLens 2 or Mixed Reality headsets using the Mixed Reality OpenXR plugin.
```

Comment:
```suggestion
description: These sample projects showcase how to build Unity applications for HoloLens 2 or Mixed Reality headsets using the Mixed Reality OpenXR plugin.
```

![image](https://user-images.githubusercontent.com/12822425/116129904-7b9ee380-a67f-11eb-8baf-0f8f3b398a95.png)
The markdown formatting seems to require this description to all be on the same line.